neo/Network/P2P/Payloads/TransactionAttributeUsage.cs

```diff
@@ -2,6 +2,6 @@ namespace Neo.Network.P2P.Payloads
 {
     public enum TransactionAttributeUsage : byte
     {
-        Url = 0x81
+        OracleExpectedResult = 0x81
```

Comment:
I think it's request, not a "result".

```suggestion
        OracleRequest = 0x81
```